sources/Renderer/OpenGL/Platform/Linux/LinuxGLContextWayland.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * LinuxGLContext.h
+ *
+ * Copyright (c) 2025 Lukas Hermanns. All rights reserved.
+ * Licensed under the terms of the BSD 3-Clause license (see LICENSE.txt).
+ */
+
+
+#if LLGL_LINUX_ENABLE_WAYLAND
```

Comment:
The header guard should always come first. Ideally we should exclude the entire file in the CMake script when Wayland support is disabled, but we can leave it to this preprocessor directive for now.